src/core_modules/capture-core/reducers/descriptions/possibleDuplicates.reducerDescription.js

```diff
@@ -0,0 +1,28 @@
+import { createReducerDescription } from '../../trackerRedux';
```

Comment:
Since now we wantedto reuse the duplicates I created a reducer for it that is "in isolation"